### PR TITLE
Updates documentation on arbiters.

### PR DESCRIPTION
**Pull request by rodrigovalin**
### All Submissions:

Updates documentation on arbiters, making it clear that `members` + `arbiters` is the resulting amount of mongods.

### CODE REVIEW

**Review by irajdeep — APPROVED** [review]
LGTM

File: docs/deploy-configure.md [hunk 1]
```diff
@@ -199,6 +184,9 @@ To add arbiters:
    kubectl apply -f <example>.yaml --namespace <my-namespace>
    ```
 
+The resulting Replica Set will be PSSA (Primary-Secondary-Seconday-Arbiter)
```
Comment (lsierant) [inline docs/deploy-configure.md:187]:
```suggestion
The resulting Replica Set will be PSSA (Primary-Secondary-Secondary-Arbiter)
```

File: docs/deploy-configure.md [hunk 2]
```diff
@@ -199,6 +184,9 @@ To add arbiters:
    kubectl apply -f <example>.yaml --namespace <my-namespace>
    ```
 
+The resulting Replica Set will be PSSA (Primary-Secondary-Secondary-Arbiter)
```
Comment (jwilliams-mongo) [inline docs/deploy-configure.md:187]: suggestion: avoid future tense whenever you can.
```suggestion
The resulting Replica Set has a PSSA (Primary-Secondary-Secondary-Arbiter)
```

File: docs/deploy-configure.md [hunk 3]
```diff
@@ -149,22 +149,21 @@ The value of the `spec.arbiters` field must be:
...
-  members: 3
   type: ReplicaSet
+  members: 3
   version: "4.2.7"
 ```
 
-To add arbiters:
-
-1. Edit the resource definition.
+To add arbiters, edit the resource definition.
```
Comment (jwilliams-mongo) [inline docs/deploy-configure.md:166]: consider reverting this change. I like that your construction is simpler, but removing the numbered list item here and not at https://github.com/mongodb/mongodb-kubernetes-operator/pull/970/files#diff-f78009da6728161bca1880b44e8c39f4d02497f945c161e525ae024a68935c11R182 leaves only a single numbered list item here that starts with `2`.
```suggestion
To add arbiters: 

1. Edit the resource definition.
```

**Review by jwilliams-mongo — APPROVED** [review]
@rodrigovalin LGTM with two small suggestions

**Pull request merged**